pkg/image/image.go

```diff
@@ -0,0 +1,77 @@
+package image
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/kustomize/v3/pkg/image"
+)
+
+func BuildRewriteList(rootDir string, host string, namespace string) ([]image.Image, error) {
+	images, err := findImages(rootDir, host, namespace, []string{})
+	return images, errors.Wrapf(err, "failed to find images")
```

Comment:
why does this return an error always?

Reply:
errors.Wrap will return nil when there is no error... it does feel a bit odd without the if